client/stylesheets/_toCat.scss

```diff
@@ -3937,7 +3939,8 @@ On click the images will be displayed at normal size to complete the effect
     height: 220px;
     display: inline-block;
     margin: 0.6%;
-    border: 2px solid #00bfff;
+    border: 1px solid #00bfff;
```

Comment:
Color literals like `#00bfff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces